### DISCUSSION TITLE
コメントフォームの「コミュニティガイドライン」のリンクをクリックできない

### DISCUSSION THREAD

**Discussion by aotoriii** [increments/qiita-discussions#560] [要望・アイデア・不具合]
10/31に修正されたコメントフォームのUIについてです。
https://qiita.com/release-notes/2023-10-31

「コミュニティガイドライン」のリンクにマウスオーバーすると、ボタンのツールチップが表示されて重なってきます。
そのせいで「コミュニティガイドライン」のリンクをクリックすることができません。
![qiita-mouse_over_tooltip](https://github.com/increments/qiita-discussions/assets/73811757/b6ca380f-1f9b-44bd-855a-63818cb69675)

PC環境のChromeで確認した事象です。ほかのブラウザは確認していません。
Windows、MacOSともに発生します。
不具合かと思いますが、ご確認をよろしくお願いします。

**Comment by tanaka-asami415** [comment]
@aotoriii 
ご報告いただきありがとうございます！
こちら不具合のため修正します🙏

**Comment by tanaka-asami415** [comment]
@aotoriii 
こちら修正が完了いたしました！ご報告いただきありがとうございました！🙇
